IPs/RSKIP208.md

```diff
@@ -0,0 +1,59 @@
+# checkEnvironment Precompile method
+
+
+|RSKIP          | 208 |
+| :------------ |:-------------|
+|**Title**      |checkEnvironment Precompile method|
+|**Created**    |JAN-2021 |
+|**Author**     |SDL |
+|**Purpose**    |Usa |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+
+# **Abstract**
+
+This RSKIP proposes to add a method `checkEnvironment()` to a new `Environment`  pre-compile contract. The new method receives as argument a list of fields that represents the minimum environment requirements to execute a method and it checks that the requirements are satisfied. This RSKIP provides the same functionality of [RKIP203](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP203.md), but in a way which allows the future addition of additional environment variables, such as rent gas limit, for storage rent accounting.
+
+## Motivation
+
+The motivation is the one described in [RSKIP203](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP203.md) plus we add the requirement that contracts that forward contracts do not become vulnerable if another counter of any kind is added to the environment.
```

Comment:
Could "contracts that forward contracts" be replaced by "contracts that forward contract calls" for clarity?

Reply:
ok